src/scoped.component.js

```diff
@@ -46,7 +43,8 @@ export class Scoped extends React.Component {
     if (
       this.props.children === undefined ||
       this.props.children === null ||
-      this.props.children === false
+      this.props.children === false ||
+      this.props.children === true
```

Comment:
`true` also indicates to react to render nothing. See https://reactjs.org/docs/react-component.html#render